Typo Fixes(PlusMigrationGuide.md) (#69)
DavidXanatos/sandboxie-docs: Content/PlusMigrationGuide.md

# Sandboxie-Plus Migration Guide

This guide shows where all the known Sandboxie functions can be found in the new UI.

## Main Window

![](../Media/Migration/0.png)

> The over all layout of the main window of sandman.exe is exactly the same as the old one of sbiectrl.exe when the "Simple View" is chosen. If the "Advanced View" is chosen, there are three additional tabs on the bottom of the window ("Sbie Messages" etc.), so it corresponds with what can be seen in the right picture.

Reviewer: ```suggestion
The overall layout of the main window of SandMan.exe is exactly the same as the old one in Sbiectrl.exe when the "Simple View" is chosen. If the "Advanced View" is chosen, there are three additional tabs on the bottom of the window ("Sbie Messages" etc.), so it corresponds with what can be seen in the right picture.
```

## File and Sandbox Menus

![](../Media/Migration/1.png)

All important menu commands can be found in similar locations, although some have been moved.

## Create New Box dialog

![](../Media/Migration/2.png)

The "Create New Box" command opens the new box dialog. Unlike in Classic, here a box type preset can be selected.

The feature to copy an existing box can be found now on another place. (See the following unit.)

## Copying Sandbox Configuration

![](../Media/Migration/3.png)

To copy a existing box configuration, the "Duplicate Sandbox" menu command can be used.

## View Menu

![](../Media/Migration/4.png)

The "View" menu  offers a few more functions, and the option to enable a simplified view mode. The recovery log is no longer a separate window but a tab at the bottom (visible when the "Advanced View" is chosen).

## View Menu - Files and Folders

![](../Media/Migration/5.png)

The modern Sandboxie UI has replaced the "Files and Folders" view with a separate window that can be opened from the box context menu.

## Files and Folders - view / window

![](../Media/Migration/6.png)

> The window "Files" offers the same functionality as the old view, but enhances it by providing a full windows context menu.

Reviewer: ```suggestion
The window "Files" offers the same functionality as the old view, but enhances it by providing a full context menu.
```

![](../Media/Migration/3.png)

> To copy a existing box configuration the "Duplicate Sandbox" menu command can be used.

Reviewer: ```suggestion
To copy a existing box configuration, the "Duplicate Sandbox" menu command can be used.
```

![](../Media/Migration/10.png)

> On this tab, the shell integration can be configured. Most functions are available, although deprecated features were dropped and other options were moved out.

Reviewer: ```suggestion
On this tab, the shell integration can be configured. Most functions are available, although some deprecated features were dropped and other options were moved out.
```

![](../Media/Migration/21.png)

Forced programs and folders are also merged into one tab ("Program Control", sub tab "Force* Programs").

## Lingering Programs & Leader Programs

![](../Media/Migration/22.png)

Program stop behaviours are also merged into one tab.

## File Migration

![](../Media/Migration/23.png)

File Migration options have been integrated into the "File Options" sub tab of the "General Options" tab.

## Internet Access

![](../Media/Migration/24.png)

Sandboxie Plus can not only use the old method of blocking internet access but also the Windows Filtering Platform (WFP), which provides better compatibility.

## Network Access

![](../Media/Migration/25.png)

> Additionally, using the WFP facility, a full per sandbox firewall can be configured (tab "Internet Restrictions" --> sub tab "Network Firewall Rules").

Reviewer: ```suggestion
Additionally, using the WFP facility, a per sandbox firewall can be configured (tab "Internet Restrictions" --> sub tab "Network Firewall Rules").
```
